php_phongo.c

```diff
@@ -1051,21 +1060,18 @@ void php_phongo_read_preference_to_zval(zval *retval, mongoc_read_prefs_t *read_
 
 void php_phongo_write_concern_to_zval(zval *retval, mongoc_write_concern_t *write_concern) /* {{{ */
 {
-	char                   *wtag;
+	const char *wtag = mongoc_write_concern_get_wtag(write_concern);
 
 	array_init_size(retval, 5);
 
-	if (mongoc_write_concern_get_w(write_concern) == MONGOC_WRITE_CONCERN_W_DEFAULT) {
-		return;
```

Comment:
I changed this to omit `w` if it's equal to `MONGOC_WRITE_CONCERN_W_DEFAULT`. We'll still print `wtag`, `"majority"`, or the numeric value for all other cases.

I'd like to do the same for `journal` and `fsync`, but that depends on mongodb/mongo-c-driver#194 being merged. As for `wtimeout`, 0 is considered the empty value for that. Ultimately, I think we need support for `null` arguments to WriteConcern or to simply take an `$options` array and be done with it. One argument is that we're locking ourselves into having an argument for `fsync`, which we already know is deprecated.